public/visualization.component.ts

```diff
@@ -1,5 +1,5 @@
 /*
- * Based on designs by Ashwin Ramaswami and Cody Leff. This file handles the "View" feature of the website. The Dots feature displays entries
+ * Based on designs by Ashwin Ramaswami and Cody Leff. This file handles the "View" feature of the website. The Travelers feature displays entries
```

Comment:
@ryanctan I don't know if we need names here (we already have a commit history), but if you're going to put names, you should also add Ryan Tan